versions.sh

```diff
@@ -88,7 +88,7 @@ for version in "${versions[@]}"; do
 		bookworm \
 		bullseye \
 		alpine3.19 \
-		alpine3.18 \
+		alpine3.20 \
```

Comment:
This ordering is load-bearing and should be what's in https://github.com/docker-library/php/pull/1514, so I'm going to merge that one instead :bow: :heart:

Thank you for the contribution!

Reply:
thanks :+1: